easyaudit/admin_helpers.py

```diff
@@ -8,7 +8,7 @@
 except: # Django < 2.0
     from django.core.urlresolvers import reverse
 
-from django.http import HttpResponseRedirect
+from django.http import HttpResponseRedirect, HttpResponse
```

Comment:
Minor (non-blocking), but it looks like we don't need `HttpResponse` imported under the current change.